apps/site/components/PropsSection/PropsSection.tsx

```diff
@@ -1,9 +1,74 @@
 import React from 'react'
 
+import styles from './props-section.module.css'
 
-const PropsSection = () => {
+export interface RowItem {
+  name: string
+  type: string | []
+  description: string
+  default: string
+  required: boolean
+}
+
+export interface PropsSectionProps {
+  propsList?: RowItem[]
```

Comment:
This is optional because we use this component on others documentations without passing this prop right? So, it would be good to add a TODO comment to turn this prop required when we finish the others.

Reply:
Nice one!